container/layers.bzl

```diff
@@ -142,7 +142,7 @@ def incremental_load(ctx, images, output,
     ]
     if run:
       run_statements += [
-          "docker run %s %s \"$@\"" % (run_flags, tag_reference)
+          "docker run %s %s $*" % (run_flags, tag_reference)
```

Comment:
I believe you should revert this line per: https://stackoverflow.com/questions/1695819/pass-bash-script-parameters-to-sub-process-unchanged

Reply:
This way all parameters will be passed as single argv.

e.g. script 1.sh
```bash
echo $1
```
```
→ bash 1.sh "1 2 3"
1 2 3
0s tmp
→ bash 1.sh 1 2 3
1
```